src/roles/nuh-deploy/tasks/main.yml

```
@@ -15,6 +15,24 @@
     ssh_host: "{{ hostname }}"
     host_username: "root"
 
+- name: Check if ha proxy is running
```

Please add "(ignoring errors)" to name to allow to be filtered out of error checking

---

done